src/styles/components/_footer.scss

```diff
@@ -22,6 +22,7 @@
 
   // Flex items of content wrapper
   &__item {
+    color: var(--foreground-color);
```

Comment:
**Current site:**
![Screen Shot 2022-10-31 at 9 44 12 AM](https://user-images.githubusercontent.com/43832671/199035686-6ea58d27-8cec-4c66-82b5-56973efec7dd.png)
![Screen Shot 2022-10-31 at 9 44 05 AM](https://user-images.githubusercontent.com/43832671/199035688-9b2a25da-08b1-482d-a36f-e608e53d4522.png)

The dark mode theme in figma shows that this should have a dark background with white text and icons.

**Figma:**
![Screen Shot 2022-10-31 at 9 45 31 AM](https://user-images.githubusercontent.com/43832671/199035910-8899529f-6c14-48ec-b7a9-50a66024f814.png)